src/daemon/mod.rs

```diff
@@ -709,14 +708,168 @@ fn create_password(prompt: &str) -> std::io::Result<String> {
         .interact_on(&term)
 }
 
+pub async fn open_forest_car_union_db(
```

Comment:
Is this method unit-tested?

Reply:
`test_prepare_and_open_forest_car_union_db` added

Reply:
And then removed, as this function no longer exists